tests/foreman/api/test_reporttemplates.py

```diff
@@ -707,3 +711,98 @@ def test_positive_generate_job_report(setup_content, target_sat, rhel7_contentho
     )
     assert res[0]['Host'] == rhel7_contenthost.hostname
     assert '/root' in res[0]['stdout']
+
+
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
+@pytest.mark.no_containers
+def test_positive_installable_errata(
+    module_org, module_location, module_cv, module_lce, rhel_contenthost, module_target_sat
+):
+    """Generate an Installable Errata report
+
+    :id: 6263a0fa-5021-4553-939b-84fb71c81d59
+
+    :setup: A Host with some applied errata
+
+    :steps:
+        1. Downgrade a package contained within the errata
+        2. Generate an Installable Errata report
+
+    :expectedresults: A report is generated with the installable errata listed
+
+    :CaseImportance: Medium
+
+    :BZ: 1726504
+    """
+    activation_key = module_target_sat.api.ActivationKey(
+        environment=module_lce, organization=module_org
+    ).create()
+    ERRATUM_ID = str(settings.repos.yum_6.errata[2])
+    module_target_sat.cli_factory.setup_org_for_a_custom_repo(
+        {
+            'url': settings.repos.yum_9.url,
+            'organization-id': module_org.id,
+            'content-view-id': module_cv.id,
+            'lifecycle-environment-id': module_lce.id,
+            'activationkey-id': activation_key.id,
+        }
+    )
+    result = rhel_contenthost.register(
+        module_org, module_location, activation_key.name, module_target_sat
+    )
+    assert f'The registered system name is: {rhel_contenthost.hostname}' in result.stdout
+    assert rhel_contenthost.subscribed
+    result = rhel_contenthost.run(f'yum install -y {FAKE_2_CUSTOM_PACKAGE}')
+    assert result.status == 0
+    rhel_contenthost.add_rex_key(satellite=module_target_sat)
```

Comment:
should not be needed if you have used `register` method already. `setup_remote_execution` is set to `True` by default

Reply:
I'm not sure if I fully understand, would the FAKE_2 pkg already be installed after the content host is registered?

Reply:
Nope, it would not. IMHO Ondrej wanted to say you don't need this line `rhel_contenthost.add_rex_key(satellite=module_target_sat)`, because it's done through `rhel_contenthost.register(...)` few lines higher by default.

Reply:
That is right, thank you both for pointing it out